contrib/asff.tpl

```diff
@@ -0,0 +1,69 @@
+[
+{{- $t_first := true -}}
+{{- range . -}}
+{{- $target := .Target -}}
+{{- range .Vulnerabilities -}}
+{{- if $t_first -}}
+  {{- $t_first = false -}}
+{{- else -}}
+  ,
+{{- end -}}
+{{- $trivyProductSev := 0 -}}
+{{- $trivyNormalizedSev := 0 -}}
+{{- if eq .Severity "LOW" -}}
+    {{- $trivyProductSev = 1 -}}
+    {{- $trivyNormalizedSev = 10 -}}
+  {{- else if eq .Severity "MEDIUM" -}}
+    {{- $trivyProductSev = 4 -}}
+    {{- $trivyNormalizedSev = 40 -}}
+  {{- else if eq .Severity "HIGH" -}}
+    {{- $trivyProductSev = 7 -}}
+    {{- $trivyNormalizedSev = 70 -}}
+  {{- else if eq .Severity "CRITICAL" -}}
+    {{- $trivyProductSev = 9 -}}
+    {{- $trivyNormalizedSev = 90 -}}
+{{- end }}
+    {
+        "SchemaVersion": "2018-10-08",
+        "Id": "{{ $target }}/{{ .VulnerabilityID }}",
+        "ProductArn": "arn:aws:securityhub:{{ getEnv "AWS_REGION" }}::product/aquasecurity/aquasecurity",
+        "GeneratorId": "Trivy",
+        "AwsAccountId": "{{ getEnv "AWS_ACCOUNT_ID" }}",
+        "Types": [ "Software and Configuration Checks/Vulnerabilities/CVE" ],
+        "CreatedAt": "{{ getCurrentTime }}",
+        "UpdatedAt": "{{ getCurrentTime }}",
+        "Severity": {
+            "Product": {{ $trivyProductSev }},
+            "Normalized": {{ $trivyNormalizedSev }}
+        },
+        "Title": "Trivy found a vulnerability to {{ .VulnerabilityID }} in container {{ $target }}",
+        "Description": {{ escapeString .Description | printf "%q" }},
+        "Remediation": {
+            "Recommendation": {
+                "Text": "More information on this vulnerability is provided in the hyperlink",
+                "Url": "{{ index .References 0 }}"
+            }
+        },
+        "ProductFields": { "Product Name": "Trivy" },
+        "Resources": [
+            {
+                "Type": "Container",
+                "Id": "{{ $target }}",
+                "Partition": "aws",
+                "Region": "{{ getEnv "AWS_REGION" }}",
+                "Details": {
+                    "Container": { "ImageName": "{{ $target }}" },
+                    "Other": {
+                        "CVE ID": "{{ .VulnerabilityID }}",
+                        "CVE Title": {{ .Title | printf "%q" }},
+                        "Installed Package": "{{ .PkgName }} {{ .InstalledVersion }}",
+                        "Patched Package": "{{ .PkgName }} {{ .FixedVersion }}"
```

Comment:
```suggestion
                        "CveID": "{{ .VulnerabilityID }}",
                        "Title": {{ .Title | printf "%q" }},
                        "PkgName": "{{ .PkgName }}"
                        "InstalledVersion": "{{ .InstalledVersion }}",
                        "FixedVersion": "{{ .FixedVersion }}"
```

Also, could you add `NvdCvssScoreV3`, `NvdCvssVectorV3`, `NvdCvssScoreV2` and `NvdCvssVectorV2`